### ISSUE TITLE
Sprite Class

### ISSUE THREAD

**Issue by drnikki** [drnikki/text-ile#76]
**Is your feature request related to a problem? Please describe.**
We seem to need to do the same thing to sprites now that we're laying them out.  In the future (aka, maybe when the term is over...), we can make a sprite class that all of our sprites extend, and that share a set of methods we seem to be continually recreating.  

**Describe the solution you'd like**
A sprite class that has some of the following methods & or properties: (developer's choice)
- align (L, R, Center)
- alignment precision or range (like, do we want it all the way left, or leftish within a range)
- mirror (print the exact same sprite, just "facing" the other direction)
- flip (print the exact same sprite, just "upside down")
- fill margins (fill visible margins with a character rather than whitespace, which character)

**Additional context**
In other words, we could do some things like (here are some random examples)
```
class Cloud extends Sprite ;

cloud.setAlign(center);
cloud.print().flip().fillMargins("-")

```

I think in order to get the flipping to work nicely, there might also need to be a standard way to write sprites, maybe? so the class would give guidance on that

**Testing for success**
Everything works exactly how it does now on the surface. :)
